decidim-core/app/models/decidim/searchable_rsrc.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "pg_search"
+
+module Decidim
+  # A searchable Resource
```

Comment:
Can you improve these docs? Explain why it's needed, how to create or some attribute names